package.json

```diff
@@ -33,7 +33,7 @@
     "format": "prettier --write \"src/**/*.js\""
   },
   "dependencies": {
-    "open": "^6.4.0"
+    "open": "^7.0.2"
```

Comment:
Any reason not to use the latest v7.0.3 release?

https://github.com/sindresorhus/open/releases/tag/v7.0.3

Reply:
Not really (although it would install the latest anyway given the caret), in fact I have just updated it! Good catch :D